components/dialog/dialog-mixin.js

```diff
@@ -14,6 +14,8 @@ import { tryGetIfrauBackdropService } from '../../helpers/ifrauBackdropService.j
 window.D2L = window.D2L || {};
 window.D2L.DialogMixin = window.D2L.DialogMixin || {};
 
+// while implemented in Webkit, native <dialog> focus mangement across slotted content is buggy
+// https://bugs.webkit.org/show_bug.cgi?id=233320
```

Comment:
I was unable to get native dialogs working in Safari because of this issue ^. I applied some fixes (coming in a separate PR) to the styling so that the original problem was resolved, but focus management is still buggy. So we'll need to live with this opt-out for a bit.